libobs/obs-output.c

```diff
@@ -792,22 +792,35 @@ size_t obs_output_get_mixers(const obs_output_t *output)
 		       : 0;
 }
 
-void obs_output_remove_encoder(struct obs_output *output,
-			       struct obs_encoder *encoder)
+void obs_output_remove_encoder_unchecked(struct obs_output *output,
```

Comment:
The going procedure for un-exported functions like this is `obs_output_remove_encoder_internal`

Reply:
k, I'll fix that up

Reply:
Fixed in 6f2f1567e1e68e1b054a6e5e649fa6a32e64dc7f